src/ogma/features/nodeGrouping.ts

```diff
@@ -71,14 +71,16 @@ export class NodeGroupingTransformation {
           const categories = new Set(nodes.getData('categories').flat());
           return {
             data: {
-              categories: Array.from(categories),
-              properties: {
-                collapsed: this._collapsedDefaultValue
-              },
+              categories: [],
+              subCategories: Array.from(categories),
+              collapsed: this._collapsedDefaultValue,
               nodeGroupId: this._findNodeGroupId(nodes)
             }
           };
         },
+        onGroupUpdate: async (_, subNodes) => {
+          return await this.runSubNodesLayout(subNodes);
```

Comment:
This will ensure that the layout runs on the subnode when the group is updated.
This is the reason why logic handling running the layout on the subnodes was removed from the front in the PR https://github.com/Linkurious/linkurious-frontend/pull/6100/files